dev/tidb-in-kubernetes/reference/configuration/storage-class.md

```diff
@@ -0,0 +1,126 @@
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl patch pvc -n <namespace> <pvc-name> -p '{"spec": {"resources": {"requests": {"storage": "100Gi"}}}'
+    ```
+
+2. View the PV size
+
+    After the expansion, the size displayed by `kubectl get pvc -n <namespace> <pvc-name>` is still the initial one, but viewing the PV size shows that it has been expanded to the expected size.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get pv | grep <pvc-name>
+    ```
+
+## Local PV Configuration
+
+Kubernetes currently supports statically allocated local storage. To create a local storage object, use the `local-volume-provisioner` program in [local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) project. The procedures are as follows:
+
+1. Pre-allocate local storage in TiKV cluster nodes. See the [operation document](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md) provided by Kubernetes for reference.
+
+2. Install the `local-volume-provisioner` program. See the [Helm installation procedure](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm) for reference.
+
+For more information, refer to [Kubernetes local storage](https://kubernetes.io/docs/concepts/storage/volumes/#local) and [local-static-provisioner document](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview).
+
+### Best Practices
+
+- The path of local PV is the unique identifier of local storage on the node. To avoid conflict, it is recommended to use the UUID of the device to generate a unique path.
+- For IO isolation, a whole physical disk per volume is recommended.
+- For capacity isolation, a separate partition per volume is recommended.
+
+Refer to [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) for more information with local PV in Kubernetes.
+
+## Data Security
+
+In general, after the PVC is used up and deleted, the PV bound to it is reclaimed and placed in the resource pool for scheduling by the provisioner. To avoid accidental data loss, you can globally configure `StorageClass`'s reclaim policy to `Retain` or only change a single PV's reclaim policy to `Retain`. In `Retain` mode, PV is not automatically reclaimed.
+
+- Global configuration
+
+    The `StorageClass` reclaim policy cannot be modified once it is created, so it can only be set at creation time. If it is not set when created, you can create another `StorageClass` of the same provisioner. For example, the default reclaim policy of `StorageClass` on Google Kubernetes Engine (GKE) is `Delete`. You can create another `StorageClass` named `pd-standard` with reclaim policy as `Retain`, and change the `storageClassName` of the corresponding component to `pd-standard` when creating TiDB clusters.
```

Comment:
```suggestion
    The `StorageClass`'s reclaim policy cannot be modified once it is created, so it can only be set at creation time. If it is not set when created, you can create another `StorageClass` of the same provisioner. For example, the default reclaim policy of `StorageClass` on Google Kubernetes Engine (GKE) is `Delete`. You can create another `StorageClass` named `pd-standard` with its reclaim policy as `Retain`, and change the `storageClassName` of the corresponding component to `pd-standard` when creating a TiDB cluster.
```